frontstage/controllers/party_controller.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from frontstage import app
+
+
+def get_respondent_by_email(email):
+
+    url = f'{app.config["PARTY_SERVICE_URL"]}/party-api/v1/respondents/email'
+    response = requests.get(url, json={"email": email}, auth=(app.config['PARTY_SERVICE_USERNAME'],
+                                                              app.config['PARTY_SERVICE_PASSWORD']))
+
+    if response.status_code == 404:
+        return 'Respondent not found'
```

Comment:
It might be better practice to raise a (custom) NotFoundError exception here instead of returning and comparing to a string here.

Reply:
@jcox-dev this shouldn't be redirecting to the error page it just shouldn't let the person log in

Reply:
@gemmai95 Sorry, I should have been more explicit in the example and said you could raise a (custom) NotFoundError exception in the controller and have the view handle (try/except block) it. (In place of the string comparison).